pym/portage/sync/modules/git/git.py

```diff
@@ -130,3 +130,15 @@ def update(self):
 			cwd=portage._unicode_encode(self.repo.location))
 
 		return (os.EX_OK, current_rev != previous_rev)
+
+	def retrieve_head(self, **kwargs):
+		'''Get information about the head commit'''
+		if kwargs:
+			self._kwargs(kwargs)
```

Comment:
I guess we don't really have to call self._kwargs if we use the repo keyword argument directly, but self._kwargs could still be useful if we needed to call any other methods that access self.repo.